src/main/steps/common/page.njk

```diff
@@ -60,6 +60,12 @@
         {% endblock %}
       {% endif %}
 
+      {% if not getContent(form.hideContactHelpSection) %}
+        {% block contactUs %}
+          {% include "common/form/contactForHelp.njk"%}
+        {% endblock %}
+      {% endif %}
+
       {# TODO uncomment this once we have contact us content #}
```

Comment:
Whilst you are here, can you remove this commented code. I think this was left over from old contact us help
Also remove it's corresponding njk file `contact-us.njk`